bower.json

```diff
@@ -43,7 +43,9 @@
     "smart-app-banner": "78ef9c0679723b25be1a0ae04f7b4aef7cbced4f",
     "habitica-markdown": "1.2.2",
     "pusher-js-auth": "^2.0.0",
-    "pusher-websocket-iso": "pusher#^3.1.0"
+    "pusher-websocket-iso": "pusher#^3.1.0",
+    "ng-tags-input": "^3.1.1",
```

Comment:
We still have a while, imo. But, I can remove this ng dependency. I don't need it anymore. The taggle is requried - yet it is a js/jquery plugin.